src/main/java/com/comphenix/protocol/utility/MinecraftReflection.java

```diff
@@ -514,7 +514,7 @@ public static Class<?> getGameProfileClass() {
 	 */
 	public static Class<?> getEntityClass() {
 		try {
-			return getMinecraftClass("server.level.Entity", "server.level.ServerEntity", "Entity");

```

Comment:
is it ever going to be server.level.[Server]Entity in one of the forks?

Reply:
No, ServerEntity is EntityTrackerEntity in spigot mappings. We only want `net/minecraft/world/entity/Entity` which is the same in mojang/spigot mappings in 1.17+ and in old versions was `net/minecraft/[pkg version]/Entity`. 

Normal paper/spigot forks won't mess with this, forge/fabric hybrid servers might be using srg/intermediary names but they usually remap reflection anyways and aren't going to do something like swap the names of Entity and ServerEntity (also special casing for hybrids in this PR where it wasn't done before seems out of scope anyways).